src/components/verification-grid/verification-grid.ts

```diff
@@ -903,10 +903,21 @@ export class VerificationGridComponent extends AbstractComponent(LitElement) {
     return !this.isViewingHistory() && !this.hasOutstandingVerification() && !this.hasOutstandingClassification();
   }
 
-  private requiredTags(): Tag[] {
+  private requiredClassificationTags(): Tag[] {
     return this.classificationDecisionElements
       .map((element) => element.tag)
-      .filter((tag, index, self) => self.indexOf(tag) === index);
+      .filter((tag, index, self) => self.findIndex((t) => t.text === tag.text) === index);
```

Comment:
~what is happening here?~

Double loop for finding uniques is super unoptimized. You can key a map with the tag text as the key and decision as a value.

We could also cache the output of this method (invalidated on slot change).